code/game/gamemodes/heist/heist.dm

```diff
@@ -56,103 +53,79 @@ var/global/list/obj/cortical_stacks = list() //Stacks for 'leave nobody behind'
+	raid_objectives = forge_raider_objectives()
 
-	var/index = 1
-
-	//Spawn the vox!
+	//Suit up the raider
 	for(var/datum/mind/raider in raiders)
 
-		if(index > raider_spawn.len)
-			index = 1
-
-		raider.current.loc = raider_spawn[index]
-		index++
-
-		create_vox(raider)
-		greet_vox(raider)
+		make_raider(raider)
+		greet_raider(raider)
 
 		if(raid_objectives)
 			raider.objectives = raid_objectives
 
 	return ..()
 
-/datum/game_mode/proc/create_vox(var/datum/mind/newraider)
+/datum/game_mode/proc/make_raider(var/datum/mind/newraider)//raiders KEEP their current species, makes for more interesting stuff
+	if(!ishuman(newraider.current))
+		to_chat(world, "VERY WWRONG PLS HALPPPPP")
+		return 0
+	var/mob/living/carbon/human/M = newraider.current
+	var/datum/preferences/P = new
+	P.species = newraider.current.client.prefs.species
+	P.real_name = M.generate_name()
+	P.random_character()
+	P.copy_to(M)
+	newraider.name = M.real_name
+
+	M.underwear = "Nude"
+	M.undershirt = "Nude"
+	M.socks = "Nude"
 
-	var/sounds = rand(2,8)
-	var/i = 0
-	var/newname = ""
+	// Do the initial caching of the player's body icons. IS THIS REALLY NEEDED
+	M.force_update_limbs()
+	M.update_dna()
+	M.update_eyes()
 
-	while(i<=sounds)
-		i++
-		newname += pick(list("ti","hi","ki","ya","ta","ha","ka","ya","chi","cha","kah"))
-
-	var/mob/living/carbon/human/vox = newraider.current
-	var/obj/item/organ/external/head/head_organ = vox.get_organ("head")
-
-	vox.real_name = capitalize(newname)
-	vox.dna.real_name = vox.real_name
-	vox.name = vox.real_name
-	newraider.name = vox.name
-	vox.age = rand(12,20)
-	vox.set_species("Vox")
-	vox.s_tone = rand(1, 6)
-	vox.languages = list() // Removing language from chargen.
-	vox.flavor_text = ""
-	vox.add_language("Vox-pidgin")
-	vox.add_language("Galactic Common")
-	vox.add_language("Tradeband")
-	head_organ.h_style = "Short Vox Quills"
-	head_organ.f_style = "Shaved"
-	vox.change_hair_color(97, 79, 25) //Same as the species default colour.
-	vox.change_eye_color(rand(1, 255), rand(1, 255), rand(1, 255))
-	vox.underwear = "Nude"
-	vox.undershirt = "Nude"
-	vox.socks = "Nude"
-
-	// Do the initial caching of the player's body icons.
-	vox.force_update_limbs()
-	vox.update_dna()
-	vox.update_eyes()
-
-	for(var/obj/item/organ/external/limb in vox.organs)
-		limb.status &= ~(ORGAN_DESTROYED | ORGAN_ROBOT)
-
-	//Now apply cortical stack.
-	var/obj/item/weapon/implant/cortical/I = new(vox)
-	I.imp_in = vox
-	I.implanted = 1
-	cortical_stacks += I
-
-	vox.equip_vox_raider()
-	vox.regenerate_icons()
 
-/datum/game_mode/proc/is_raider_crew_safe()
-	if(cortical_stacks.len == 0)
-		return 0
 
-	for(var/obj/stack in cortical_stacks)
-		if(get_area(stack) != locate(/area/shuttle/vox) && get_area(stack) != locate(/area/vox_station))
-			return 0 //this is stupid as fuck
-	return 1
+	M.equip_raider()
+	M.rejuvenate()//removes people being stupid and taking limbless, mute and blind characters, also regenrates icons.
```

Comment:
this should be impossible, you create a new pref datum, that should prevent limbless/mute/blind characters